pkg/consts/consts.go

```diff
@@ -53,3 +58,21 @@ const (
 	// InstanceSettingsID is the id of settings document for the instance
 	InstanceSettingsID = "io.cozy.settings.instance"
 )
+
+const (
+	// OneShotSharing is a sharing with no continuous updates
+	OneShotSharing = "one-shot"
+	// MasterSlaveSharing is a sharing with unilateral continuous updates
+	MasterSlaveSharing = "master-slave"
+	// MasterMasterSharing is a sharing with bilateral continuous updates
+	MasterMasterSharing = "master-master"
+)
+
+const (
+	// PendingStatus is the sharing pending status
+	PendingStatus = "pending"
+	// RefusedStatus is the sharing refused status
+	RefusedStatus = "refused"
+	// AcceptedStatus is the sharing accepted status
+	AcceptedStatus = "accepted"
```

Comment:
Ok, I'll rename in `PendingSharingStatus`, I prefer to keep everything in one place for now.